changelogs/fragments/licenses.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "All software licenses are now in the ``LICENSES/`` directory of the collection root (https://github.com/ansible-collections/community.crypto/pull/491)."
```

Comment:
This looks like a mistake

Reply:
It definitely is! Sometimes `git stash push/pop` is confusing me... I was working on a licensing PR, then got interrupted for some time, and forgot about it when starting on this one... unti noticing that there were waaaay to many changes. Let me fix that.